### PR TITLE
Adds load away mission verb

### PR DESCRIPTION
**Pull request by AnturK**
Also splits whitelist from roundstart loading config.

### CODE REVIEW

File: config/game_options.txt [hunk 1]
```diff
@@ -270,6 +270,9 @@ ALLOW_AI
 
 ## AWAY MISSIONS ###
 
+# Will random away mission be loaded at initialization.
```
Comment (ExcessiveUseOfCobblestone) [inline config/game_options.txt]: do what now

Comment (AnturK) [reply config/game_options.txt]: What's the problem ?

Comment (PKPenguin321) [reply config/game_options.txt]: should have a ? at the end, with the period it makes it read like a statement

Comment (Cyberboss) [inline config/game_options.txt]: Needs another #

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Pull request merged**